crates/runtime/src/backtrace/s390x.rs

```diff
@@ -0,0 +1,19 @@
+pub unsafe fn get_next_older_pc_from_fp(fp: usize) -> usize {
+    *(fp as *mut usize).offset(14)
+}
+
+pub unsafe fn get_next_older_fp_from_fp(fp: usize) -> usize {
+    *(fp as *mut usize)
+}
+
+pub fn reached_entry_sp(fp: usize, first_wasm_sp: usize) -> bool {
+    fp == first_wasm_sp
+}
```

Comment:
Hi @uweigand, would you mind suggesting wording for some comments for these functions similar to those in the `x86_64.rs` file just below? I'm not familiar enough to add the docs that Alex requested in review myself.

Reply:
get_next_older_pc_from_fp:
>The next older PC can be found in register %r14 at function entry, which was saved into slot 14 of the register save area pointed to by 'FP' (the backchain pointer).

get_next_older_fp_from_fp:
>The next older 'FP' (backchain pointer) was saved in the slot pointed to by the current 'FP'.

reached_entry_sp:
>The 'FP' (backchain pointer) holds the value of the stack pointer at function entry. If this equals the value the stack pointer had when we first entered a Wasm function, we are done.

Reply:
Not much to say about the stack alignment except that the ABI guarantees 8-byte alignment at all times, and we cannot in general expect anything more.

Reply:
Thanks!